tests/kuttl/test-suites/default/watcher/01-assert.yaml

```diff
@@ -20,6 +20,7 @@ spec:
   apiServiceTemplate:
     replicas: 1
     resources: {}
+  prometheusSecret: metric-storage-prometheus-config
```

Comment:
so in addtion to this the metric-storage-prometheus-config secret should now have a
finaliser form the watcher operator.